Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Christopher Loggins</authors>
+        <owners>Christopher Loggins</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.0" />
```

Comment:
It would. Even if, as you said, FIE 3 works w/ this library. That's why I went with the >=2.0 setting, but no upper limit.